PED/SetPedShouldPlayDirectedScenarioExit.md

```diff
@@ -6,13 +6,20 @@ aliases: ["0xEC6935EBE0847B90"]
 
 ```c
 // 0xEC6935EBE0847B90 0x761F8F48
-Any _SET_PED_SHOULD_PLAY_DIRECTED_SCENARIO_EXIT(Any p0, Any p1, Any p2, Any p3);
+BOOL _SET_PED_SHOULD_PLAY_DIRECTED_SCENARIO_EXIT(PED_INDEX PedIndex, float x, float y, float z);
```

Comment:
Please use `camal case` too.

Real native name seems to be `SET_PED_SHOULD_PLAY_DIRECTED_NORMAL_SCENARIO_EXIT` -- I'll let @4mmonium see, im not sure.